check_docker.py

```diff
@@ -224,9 +224,10 @@ def process_args(args):
 
     # Container name
     parser.add_argument('--containers',
+                        required=True,
```

Comment:
I get why you are requiring '--containers' but I think you are missing a use case. I envision want to install this on a server with lots of container workloads(maybe a utility box) which may a lot of different containers or ones that change frequently. By defaulting to 'all' we can monitor whatever workloads there are without having to right a check for each.  

As a result of this I would rather not merge this improvement.